Composer/packages/client/src/components/CreationFlow/index.tsx

```diff
@@ -44,8 +44,10 @@ const CreationFlow: React.FC<CreationFlowProps> = () => {
   }, [storages]);
 
   const init = async () => {
-    fetchTemplates();
-    fetchStorages();
+    (async function() {
+      await fetchStorages();
+      await fetchTemplates();
+    })();
   };
```

Comment:
Is there a reason this can't just be:
```
const init = async () => {
      await fetchStorages();
      await fetchTemplates();
  };
```
?

Is wrapping it in an extra layer of async needed somewhere?

Reply:
Yep these will be reverted. They were made just to check if an Integration test passed with this change.